prince-api.js

```diff
@@ -199,6 +202,14 @@ Prince.prototype.inputs = function (inputs) {
     return this;
 };
 
+/* set cookies */
+Prince.prototype.cookies = function (cookies) {
+    if (arguments.length !== 1)
+        throw new Error("Prince#inputs: invalid number of arguments");
```

Comment:
This should read "Prince#cookies:", of course.

Reply:
Indeed. Definitely not my day.